### DISCUSSION TITLE
stopped getting mobile notifications

### DISCUSSION THREAD

**Discussion by daniellperetz** [WhiskeySockets/Baileys#83] [General Questions]
When running this code as a bot script and using my own phone number(as in my mobile whatsapp), I do not get sound alerts on my phone, the same thing as opening whatsapp web on pc.
Is there a way to make this script kind of inactive/minimized to get messages on my phone and still listen to messages?

**Comment by mfirdausazizi** [comment]
Set markOnlineOnConnect as false and notification should be working fine

**Reply by leandrodesign** [reply]
it didn't work for me 😔

class WhatsAppInstance {
    socketConfig = {
        defaultQueryTimeoutMs: undefined,
        markOnlineOnConnect: false,
        printQRInTerminal: false,
        logger: pino({
            level: config.log.level,
        }),
    }

**Comment by salomonamiga** [comment]
Hello, did you have a solution for this ?

**Comment by Riders004** [comment]
> When running this code as a bot script and using my own phone number(as in my mobile whatsapp), I do not get sound alerts on my phone, the same thing as opening whatsapp web on pc. Is there a way to make this script kind of inactive/minimized to get messages on my phone and still listen to messages?

I'm not getting yet

**Comment by marcospaulo-4** [comment]
put this in your code after some action, for example sending a message, the sound notifications will start working again, for me it worked

`sock.sendPresenceUpdate('unavailable')`
